.github/workflows/projectboard.yml

```diff
@@ -20,12 +20,4 @@ jobs:
           echo $json
           echo "::set-output name=matrix::$(echo $json)"
     outputs:
-      issueTags: ${{ steps.set-matrix.outputs.matrix }}
-      
-  Manage_project_issues:
```

Comment:
Not needed for us. And even then, i replaced this project board CI and Penny will be handling these, and this will be removed from the vapor too.